### PR TITLE
Updated navigation to include links for adding linter provider instructions

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Closing the loop for https://github.com/AtomLinter/atomlinter.github.io/issues/31

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -19,5 +20,6 @@
     {% for scope in site.data.providers %}
     <a class="mdl-navigation__link" href="#{{ scope.modal }}">{{ scope.title }}</a>
     {% endfor %}
+    <a class="mdl-navigation__link" href="https://github.com/AtomLinter/atomlinter.github.io#adding-a-linter-provider">Add your Linter Provider</a>
```
Comment (Arcanemagus) [inline _includes/nav.html]: Shouldn't this link start with `//` like the others?

File: _includes/nav.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <div class="mdl-layout-spacer"></div>
     <!-- Navigation. We hide it in small screens. -->
     <nav class="mdl-navigation mdl-layout--large-screen-only">
+      <a class="mdl-navigation__link" href="//github.com/AtomLinter/atomlinter.github.io#adding-a-linter-provider">Add your Linter Provider</a>
```
Comment (Arcanemagus) [inline _includes/nav.html]: This text fits in on the drawer, but up here in the navigation bar it's pretty large, is there a shorter wording that will get the same point across?

image

**Comment by gaurav-nelson** [comment]
@Arcanemagus I was away for a while. Fixed as per you suggestions. Please take a look.

**Comment by keplersj** [comment]
Looks awesome.
